sql/_05_plcsql/_01_testspec/_05_bug_fix/cases/02_column_name.sql

```diff
@@ -0,0 +1,16 @@
+--+ server-message on
+-- Verified for CBRD-24946
+-- Field lookup failure when column name is of <table>.<column> form in the select list
+
+create or replace procedure test_proc as
+begin
+    for r in (select t.name from db_user t) loop
```

Comment:
select t.name from db_user t order by t.name

Reply:
Thx, I reflect that.